txscript/data/script_tests.json

```diff
@@ -1365,87 +1459,193 @@
-["11 10", "LESSTHANOREQUAL NOT", "NONE", "OK"],
+["1 2147483648", "GREATERTHAN", "NONE", "ERR_OUT_OF_RANGE", "GREATERTHAN must fail with stack item >4 bytes"],
+
+["LESSTHANOREQUAL test coverage"],
+["NOP", "LESSTHANOREQUAL", "NONE", "ERR_INVALID_STACK_OPERATION", "LESSTHANOREQUAL requires two items on the stack and NOP must not be treated as one"],
+["", "LESSTHANOREQUAL", "NONE", "ERR_INVALID_STACK_OPERATION", "LESSTHANOREQUAL requires two items on the stack, stack has no items"],
+["0", "LESSTHANOREQUAL", "NONE", "ERR_INVALID_STACK_OPERATION", "LESSTHANOREQUAL requires two items on the stack, stack has one item"],
+["11 10", "LESSTHANOREQUAL NOT", "NONE", "OK", "LESSTHANOREQUAL must check if the first numeric stack item is less than or equal to the second numeric stack item"],
 ["4 4", "LESSTHANOREQUAL", "NONE", "OK"],
 ["10 11", "LESSTHANOREQUAL", "NONE", "OK"],
 ["-11 11", "LESSTHANOREQUAL", "NONE", "OK"],
 ["-11 -10", "LESSTHANOREQUAL", "NONE", "OK"],
-["11 10", "GREATERTHANOREQUAL", "NONE", "OK"],
+["1 2147483648", "LESSTHANOREQUAL", "NONE", "ERR_OUT_OF_RANGE", "LESSTHANOREQUAL must fail with stack item >4 bytes"],
+
+["GREATERTHANOREQUAL test coverage"],
+["NOP", "GREATERTHANOREQUAL", "NONE", "ERR_INVALID_STACK_OPERATION", "GREATERTHANOREQUAL requires two items on the stack and NOP must not be treated as one"],
+["", "GREATERTHANOREQUAL", "NONE", "ERR_INVALID_STACK_OPERATION", "GREATERTHANOREQUAL requires two items on the stack, stack has no items"],
+["0", "GREATERTHANOREQUAL", "NONE", "ERR_INVALID_STACK_OPERATION", "GREATERTHANOREQUAL requires two items on the stack, stack has one item"],
+["11 10", "GREATERTHANOREQUAL", "NONE", "OK", "GREATERTHANOREQUAL must check if the first numeric stack item is greater then or equal to the second numeric stack item"],
 ["4 4", "GREATERTHANOREQUAL", "NONE", "OK"],
 ["10 11", "GREATERTHANOREQUAL NOT", "NONE", "OK"],
 ["-11 11", "GREATERTHANOREQUAL NOT", "NONE", "OK"],
 ["-11 -10", "GREATERTHANOREQUAL NOT", "NONE", "OK"],
-["1 0 MIN", "0 NUMEQUAL", "NONE", "OK"],
+["1 2147483648", "GREATERTHANOREQUAL", "NONE", "ERR_OUT_OF_RANGE", "GREATERTHANOREQUAL must fail with stack item >4 bytes"],
+
+["MIN test coverage"],
+["NOP", "MIN", "NONE", "ERR_INVALID_STACK_OPERATION", "MIN requires two items on the stack and NOP must not be treated as one"],
+["", "MIN", "NONE", "ERR_INVALID_STACK_OPERATION", "MIN requires two items on the stack, stack has no items"],
+["0", "MIN", "NONE", "ERR_INVALID_STACK_OPERATION", "MIN requires two items on the stack, stack has one item"],
+["1 0 MIN", "0 NUMEQUAL", "NONE", "OK", "MIN must compare the first two numeric stack items and return the mimimum"],
 ["0 1 MIN", "0 NUMEQUAL", "NONE", "OK"],
 ["-1 0 MIN", "-1 NUMEQUAL", "NONE", "OK"],
 ["0 -2147483647 MIN", "-2147483647 NUMEQUAL", "NONE", "OK"],
-["2147483647 0 MAX", "2147483647 NUMEQUAL", "NONE", "OK"],
+["1 2147483648", "MIN", "NONE", "ERR_OUT_OF_RANGE", "MIN must fail with stack >4 bytes"],
```

Comment:
Same here regarding needing another test for the other operand being >4 bytes and comment.